packages/procps/configure.ac.patch

```diff
@@ -0,0 +1,23 @@
+--- procps-3.3.17.orig/configure.ac	2021-02-09 17:11:25.000000000 +0700
++++ procps-3.3.17/configure.ac	2021-05-12 00:45:18.000000000 +0700
+@@ -76,10 +76,8 @@
+ # Checks for library functions.
+ AC_FUNC_ERROR_AT_LINE
+ AC_FUNC_FORK
+-AC_FUNC_MALLOC
+ AC_FUNC_MBRTOWC
+ AC_FUNC_MMAP
+-AC_FUNC_REALLOC
```

Comment:
Why does AC_FUNC_{MALLOC,REALLOC} need to be removed?

Reply:
Because it caused `undefined references to 'rpl_malloc'` in my build.